### ISSUE TITLE
[Electron/Chromium regression] Tray icon disappears after locking the screen on GNOME

### ISSUE THREAD

**Issue by zambelly** [jiahaog/nativefier#945]
**Bug description**
The tray icon disappears after locking and unlocking the screen.

- [x] I looked at `nativefier --help` and https://github.com/jiahaog/nativefier/blob/master/docs/api.md
- [x] I searched existing open & closed issues. Yes, my bug is new.
- [x] I'm using the latest version available at https://github.com/jiahaog/nativefier/releases

**Steps to reproduce**
1. Build (https://aur.archlinux.org/cgit/aur.git/tree/?h=whatsapp-nativefier):
```
nativefier \
  --verbose \
  --name "WhatsApp" \
  --icon "whatsapp-nativefier.png" \
  --width "800px" \
  --height "600px" \
  --inject "whatsapp-nativefier-inject.js" \
  --verbose \
  --single-instance \
  --tray \
  "https://web.whatsapp.com"
```
Build output: https://termbin.com/brgh

2. Launch app
3. Lock the screen
4. Unlock the screen

**Expected behavior**
See the WhatsApp tray icon remain in my system tray

**Actual behavior**
The WhatsApp tray icon disappears.

**Debug info**
Before locking & unlocking the screen:
![Screenshot from 2020-04-13 09-34-46](https://user-images.githubusercontent.com/30311816/79102439-09f14180-7d6b-11ea-9a0b-414d5a61af3a.png)
After locking & unlocking the screen:
![Screenshot from 2020-04-13 09-34-56](https://user-images.githubusercontent.com/30311816/79102505-29886a00-7d6b-11ea-9046-903f11522c66.png)

**Context**
 - Nativefier: 8.0.6
 - Node.js: 13.12.0
 - Npm: 6.14.4
 - OS: Manjaro Linux, GNOME 3.36.1
 - Is it a regression? No
 - Additional context: I started experiencing the issue I believe when I updated to GNOME 3.36. I had a similar [issue](https://github.com/telegramdesktop/tdesktop/issues/7541) with Telegram's desktop app, which is now [fixed](https://github.com/telegramdesktop/tdesktop/pull/7624)

**Comment by OgulcanCelik** [comment]
Same issue happening on me, also on Manjaro Linux, Gnome 3.36 with the same package, it disappears after sleep.

**Comment by AlanSune** [comment]
hello, can try to use this options: `e 7.2.4`.
it work well on ubuntu 20.04.

**Comment by zambelly** [comment]
As far as I can tell, `--electron-version 7.2.4` solved the issue. Thanks @AlanSune

**Comment by AlanSune** [comment]
Good news, the latest verson of electron maybe fixed this bug.
Can try to use this options: `-e 9.0.3` or `--electron-version  9.0.3`.
it work well on my ubuntu 20.04.
[Release electron v9.0.3 · electron/electron](https://github.com/electron/electron/releases/tag/v9.0.3)

**Comment by ronjouch** [comment]
> Good news, the latest verson of electron maybe fixed this bug.
> Can try to use this options: `-e 9.0.3` or `--electron-version 9.0.3`.
> It work well on my ubuntu 20.04.
> [Release electron v9.0.3 · electron/electron](https://github.com/electron/electron/releases/tag/v9.0.3)

@AlanSune thanks! I just released [Nativefier v9.0.0](https://github.com/jiahaog/nativefier/releases/tag/v9.0.0) which defaults to Electron 9.0.4. Closing, feel free to re-open as needed.
